runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/expression/EnvironmentResolverSpi.java

```diff
@@ -0,0 +1,24 @@
+package io.aklivity.zilla.runtime.engine.internal.expression;
+
+import java.util.Optional;
+import java.util.function.UnaryOperator;
+
+import io.aklivity.zilla.runtime.engine.expression.ExpressionResolverSpi;
+
+public class EnvironmentResolverSpi implements ExpressionResolverSpi
+{
+
+    @Override
+    public String name()
+    {
+        return "env";
+    }
+
+    @Override
+    public String resolve(
+        String var)
+    {
+        UnaryOperator<String> env = System::getenv;
+        return Optional.ofNullable(env.apply(var)).orElse("");
```

Comment:
Avoid object allocation of `Optional` on each call to `resolve`.
Return `null` if not found, default to `""` is happening at caller `ExpressionResolver`.
Simplify to `return System.getenv(var);`.